content/kubermatic/master/tutorials-howtos/external-clusters/aks/_index.en.md

```diff
@@ -13,15 +13,27 @@ Click `Add External Cluster` button and Pick `Azure Kubernetes Cluster` provider
 
 ![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
 
+![Select Provider](/img/kubermatic/master/tutorials/external_clusters/connect.png "Select Provider")
+
 Select preset with valid credentials or enter AKS `Tenant ID`, `Subscription ID`, `Client ID` and  `Client Secret`, to connect to the provider.
-The credentials should have enough access like read, write Azure Kubernetes Service and list cluster admin credential action, to fetch kubeconfig using API.
+The credentials should have enough access like read, write Azure Kubernetes Service and list cluster admin credential, to fetch kubeconfig using API.
 
 ![AKS credentials](/img/kubermatic/master/tutorials/external_clusters/aks_credentials.png "AKS credentials")
 
 You should see the list of all available clusters. Select the one and click the `Import Cluster` button. Clusters can be imported only once in a single project. The same cluster can be imported from the other projects.
 
 ![Select AKS cluster](/img/kubermatic/master/tutorials/external_clusters/select_aks_cluster.png "Select AKS cluster")
 
+### Provide the credentials:
+Credentials can be provided either by:
+selecting a precreated preset which stores the provider specific credentials
+manually enter the credentials
+After user provides all required credentials, credentials will be validated
+
+AKS:
+Create a preset on your KKP cluster with `spec.aks.clientID`, `spec.aks.clientSecret`,
```

Comment:
can we update order just like UI image otherwise you can skip this comment if not necessary
- `spec.aks.tenantID`.
- `spec.aks.subscriptionID
- `spec.aks.clientID`
- `spec.aks.clientSecret`